### ISSUE TITLE
Question about data processing

### ISSUE THREAD

**Issue by kkk00714** [SajjadMzf/TrajPred#5]
Hello, I would like to know some specific steps about data set processing, such as what the data set goes through before it is divided into a training set/test set/verification set, and how many size sliding Windows you use to sample the same car. Thank you very much! @SajjadMzf

**Comment by yangxingbang** [comment]
用作者的tpl库中的preprocess处理数据集，比如highd；用作者的tpl库中的feature将preprocess的结果打包成scenario和state，比如18个变量的那个state；然后就能形成训练模型的数据集。

如果你问的滑动窗口指的是输入sequence的长度，那么是40，其中输入数据15帧，时间长度是3s，输出数据25帧，时间长度5秒，合计40帧，8秒。长40的sequence将来既要作为输入数据，也要作为计算损失的真值。

**Comment by 777-q** [comment]
您好，请问这个tpl库怎末安装哇，谢谢！

**Comment by yangxingbang** [comment]
![777777](https://github.com/user-attachments/assets/cd0123ee-079a-4a9a-8d49-a65b1befb856)

**Comment by yangxingbang** [comment]
> 您好，请问这个tpl库怎末安装哇，谢谢！

回复晚了，或许以后还有人用。

**Comment by ghostwhi** [comment]
怎么把highD的csv转成训练需要的h5啊

**Comment by yangxingbang** [comment]
> 怎么把highD的csv转成训练需要的h5啊
TrajFeature_for_prediction中render_scenarios.py中def save_dataset(self):
@ghostwhi

**Comment by FirstAmyAmyAmy** [comment]
Possible map-version mismatch for exiD Location 3: missing OSM way 1519
